core/Console.php

```diff
@@ -209,6 +209,35 @@ private function getDefaultPiwikCommands()
         $commands = array(
             'Piwik\CliMulti\RequestCommand'
         );
+
+        $commandsFromPluginsMarkedInConfig = $this->getCommandsFromPluginsMarkedInConfig();
+        if(!empty($commandsFromPluginsMarkedInConfig)) {
+            $commands = array_merge($commands, $commandsFromPluginsMarkedInConfig);
+        }
+
+        return $commands;
+    }
+
+    private function getCommandsFromPluginsMarkedInConfig()
+    {
+        $general = Config::getInstance()->getFromCommonConfig('General');
```

Comment:
Any reason we are not using `$config->General['always_load_commands_from_plugin']` and force it to be in common config?